src/targets/gpu/hiprtc/CMakeLists.txt

```diff
@@ -28,6 +28,10 @@ add_executable(migraphx-hiprtc-driver
 rocm_clang_tidy_check(migraphx-hiprtc-driver)
 target_link_libraries(migraphx-hiprtc-driver PRIVATE migraphx_gpu)
 add_dependencies(migraphx_all_targets migraphx-hiprtc-driver)
-rocm_install_targets(
-    TARGETS migraphx-hiprtc-driver
-)
+
+# For ASAN Enabled Build no need to package binaries
+if( NOT ENABLE_ASAN_PACKAGING )
+  rocm_install_targets(
+      TARGETS migraphx-hiprtc-driver
```

Comment:
I think this will make it really slow for ASAN builds since we will compile serially instead of in parallel.

Reply:
In our ci the builds are done parallelly. Normal builds are done in a job. And asan builds are done later in a separate job.

Reply:
I mean when the users are compiling at runtime, this will be very slow. Its already much slower from ASAN itself, but now it will be massively slower(ie 48x).

Why cant we have this executable as part of the ASAN package?

Also, arent users interested in running everything with ASAN? If we remove this for ASAN, then there wont be ASAN coverage for this code, which is run in our release packages.